manifests/init.pp

```diff
@@ -12,25 +12,47 @@
 #
 # Sample Usage: netrc::foruser("netrc_myuser": user => 'myuser', machine_user_password_triples => [['myserver.localdomain','myuser','pw'],['mysecondserver.localdomain','myuser','pw2']])
 #               you can also override the full path by using the `file_path` parameter.
-# [Remember: No empty lines between comments and class definition]
 class netrc {
-
 }
 
-define netrc::foruser(
-  Enum["present", "absent"]     $ensure = "present",
-  Stdlib::Absolutepath          $home_base_directory  = "/home",
-  String                        $user,
-  String                        $filename             = ".netrc",
-  Stdlib::Absolutepath          $file_path            = "$home_base_directory/$user/$filename",
-  Hash[String, Hash]            $machine_login_password) {
-
-  file { $file_path:
-    ensure  => $ensure,
-    content => epp('netrc/netrc.epp', {
-      machine_login_password => $machine_login_password
-    }),
-    mode    => '0600',
-    owner   => "$user"
+define netrc::usermachine (
```

Comment:
wouldn't it be better to use the word `host` instead of `machine`?

Reply:
I used machine as it's the syntax used in netrc file.